spring-cloud-gcp-data-firestore/src/main/java/org/springframework/cloud/gcp/data/firestore/FirestoreReactiveOperations.java

```diff
@@ -27,14 +28,22 @@
  */
 public interface FirestoreReactiveOperations {
 	/**
-	 * Saves an instance of an object to Cloud Firestore.
-	 * Behaves as update or insert.
+	 * Saves an instance of an object to Cloud Firestore. Behaves as an insert only (not
+	 * update).
 	 * @param <T> the type of the object to save.
 	 * @param instance the instance to save.
 	 * @return the instance that was saved.
 	 */
 	<T> Mono<T> save(T instance);
 
+	/**
+	 * Saves multiple objects to Cloud Firestore. Behaves as insert only (not update).
```

Comment:
I think we should make it to work for both update and inserts.

Reply:
done !